plugins/modules/inventory_intent.py

```diff
@@ -2774,37 +2851,39 @@ def verify_diff_merged(self, config):
         if device_added:
             if not devices_to_add:
                 self.status = "success"
-                msg = "Requested Devices - {0} Added in Cisco DNA Center and Addition verified.".format(str(device_ips))
-                self.log(msg)
+                msg = "Requested Device(s) '{0}' Added in Cisco Catalyst Center and Addition verified.".format(str(device_ips))
```

Comment:
"Requested device(s) '{0}' have been successfully added to Cisco Catalyst Center and their addition has been verified.".format(str(device_ips))

Reply:
added the list of devices.